arduino-ide-extension/package.json

```diff
@@ -155,11 +155,7 @@
   ],
   "arduino": {
     "cli": {
-      "version": {
-        "owner": "cmaglie",
-        "repo": "arduino-cli",
-        "commitish": "small_delay_monitor_close"  
-      }
+      "version": "0.23.0"
```

Comment:
This will cause the builds to no longer have the fix from https://github.com/arduino/arduino-cli/pull/1771

I can always manually replace Arduino CLI, so it is not a problem for me, but worth noting for the sake of other testers.